src/components/OrgPeopleListCard/OrgPeopleListCard.module.css

```diff
@@ -1,74 +0,0 @@
-.memberlist {
-  margin-top: -1px;
-}
-.memberimg {
-  width: 200px;
-  height: 100px;
-  border-radius: 7px;
-  margin-left: 20px;
-}
-.singledetails {
-  display: flex;
-  flex-direction: row;
-  justify-content: space-between;
-}
-.singledetails p {
-  margin-bottom: -5px;
-}
-.singledetails_data_left {
-  margin-top: 10px;
-  margin-left: 10px;
-  color: #707070;
-}
```

Comment:
Why this file deleted ?

Reply:
Because no class in that file is being used. We are using react-bootstrap components that takes care of the styles.